### ISSUE TITLE
windows10上使用CPU训练识别模型没有生成best_accuracy文件

### ISSUE THREAD

**Issue by 18328401960** [PaddlePaddle/PaddleOCR#2157]
使用的是官方教程采用，配置文件为rec_icdar15_train.yml
数据集如下
![image](https://user-images.githubusercontent.com/60595295/109624566-54dd9880-7b79-11eb-99c8-7282e30bdda3.png)

配置如下
Global:
  use_gpu: false
  epoch_num: 72
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/rec/ic15/
  save_epoch_step: 3
  # evaluation is run every 2000 iterations
  eval_batch_step: [0, 10]
  cal_metric_during_train: True
  pretrained_model: 
  checkpoints:
  save_inference_dir:
  use_visualdl: False
  infer_img: doc/imgs_words_en/word_10.png
  # for data or label process
  character_dict_path: ppocr/utils/ic15_dict_train.txt
  character_type: ch
  max_text_length: 25
  infer_mode: False
  use_space_char: False

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    learning_rate: 0.0005
  regularizer:
    name: 'L2'
    factor: 0

Architecture:
  model_type: rec
  algorithm: CRNN
  Transform:
  Backbone:
    name: MobileNetV3
    scale: 0.5
    model_name: large
  Neck:
    name: SequenceEncoder
    encoder_type: rnn
    hidden_size: 96
  Head:
    name: CTCHead
    fc_decay: 0

Loss:
  name: CTCLoss

PostProcess:
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/ic15_data/
    label_file_list: ["./train_data/ic15_data/rec_gt_train.txt"]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 100]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: True
    batch_size_per_card: 256
    drop_last: True
    num_workers: 8
    use_shared_memory: False

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/ic15_data/
    label_file_list: ["./train_data/ic15_data/rec_gt_test.txt"]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 100]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 256
    num_workers: 4
    use_shared_memory: False

第一次epoch_num：500    eval_batch_step: [0, 100]
结果如下：
![1](https://user-images.githubusercontent.com/60595295/109624502-40010500-7b79-11eb-8b1c-409846f669ca.jpg)

iter10时自动退出了，没有往下进行

第二次epoch_num：72    eval_batch_step: [0, 10]
结果如下：
![2](https://user-images.githubusercontent.com/60595295/109624530-48594000-7b79-11eb-9873-1aa30f4c6ba9.jpg)

iter10时，评估后也自动退出
想询问下是什么原因造成的，不知道是不是配置文件有错。想寻求下帮助，非常感谢！

**Comment by littletomatodonkey** [comment]
你好，在CPU上训练的时候，建议将batch size调小一点

**Comment by 18328401960** [comment]
> 你好，在CPU上训练的时候，建议将batch size调小一点

你好，我分别将batch_size_per_card改为128、64、32分别又训练了一次，还是在iter10的时候退出了。
![image](https://user-images.githubusercontent.com/60595295/109744691-b0596600-7c0d-11eb-997f-9e7a06ca4505.png)
![image](https://user-images.githubusercontent.com/60595295/109744716-bbac9180-7c0d-11eb-84c5-0f0d0d1b9a6a.png)
![image](https://user-images.githubusercontent.com/60595295/109744737-c535f980-7c0d-11eb-9187-60139fccddb0.png)
